string eqp_cd
string result_code        # "0000","0010","0020","0030"
int32  len_result_p1_2
int32  len_result_p3_4
int32  width_result_p5_6
int32  width_result_p7_8
builtin_interfaces/Time stamp
